hack/run-e2e-tests.sh

```diff
@@ -6,28 +6,45 @@ set -o pipefail
 set -o xtrace
 
 INTEGRATION_OUTPUT_JUNIT=${INTEGRATION_OUTPUT_JUNIT:-false}
+VERSION=${VERSION:-test}
+
+docker build . \
+    --build-arg ldflags_arg="" \
+    -t "kudobuilder/controller:$VERSION"
+
+./bin/kubectl-kudo init --dry-run --output yaml \
+    | sed -E -e "/image:/ s/:[[:alnum:]]+/:$VERSION/" -e '/imagePullPolicy/ s/Always/Never/' \
+    > test/manifests/kudo.yaml
```

Comment:
Maybe we should have a file in `test/manifests/kudo.yaml` with a note that the file will be overwritten all the time? Just to prevent that somebody tries to put a static config there?

Reply:
Good point! The tests currently creates artifacts all over the place and doesn't bother with cleaning up temporaries once they're done. This is something that need to be fixed at some point, but independently from this PR. How about having a separate manifests folder for these generated files?